tasks/main.yml

```diff
@@ -13,6 +13,14 @@
     jenkins_repo_key_url: "{{ __jenkins_repo_key_url }}"
   when: jenkins_repo_key_url is not defined
 
+- name: Create Jenkins user with ssh keys
```

Comment:
This task is setting jenkins user home to jenkins_home variable which is set to /var/lib/jenkins in the playbook. Let me add it to role defaults